src/agent/libclusterfuzz/third-party/build.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+
+
+import string
+import constants
+
+data = {}
+
+name_check = string.ascii_uppercase + "_"
```

Comment:
I think you forgot to use this below, will comment there.

Also, I suggest:
```suggestion
NAME_CHECK = string.ascii_uppercase + "_"
```

Reply:
(also, maybe there's a better name for this character list?)